backend/pharmacy/api/urls.py

```diff
@@ -1,17 +1,34 @@
+from django.conf import settings
 from django.urls import path
 from . import views
 from rest_framework_simplejwt.views import (
     TokenRefreshView,
     TokenVerifyView
 )
 from .views import MyTokenObtainPairView
+from rest_framework.urlpatterns import format_suffix_patterns
+
+from django.conf.urls.static import static
+from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+
 
 urlpatterns = [
-    path('', views.getData),
-    path('post/', views.postData),
-    path('update/', views.updateData),
-    path('get/', views.getRoutes),
+    path('', views.DataList.as_view(), name='get'),
+    path('<int:pk>/', views.Data.as_view()),
```

Comment:
This one is interesting:

@rahul007-bit @Mansiviramgama @Mayank-0203 This line will allow me to get arguments through the url. 
```python
<int:pk>
```
here int is the data type of my argument and pk is the variable that is going to be used in function definition.